### ISSUE TITLE
Is snippet output configurable?

### ISSUE THREAD

**Issue by kornelijussl** [xabikos/vscode-react#77]

**Comment by rdaraujo** [comment]
Yes, you can find the templates in this file `~/.vscode/extensions/xabikos.reactsnippets-X.Y.Z/snippets/snippets.json` (pay attention to the version X.Y.Z).
